woocommerce/woocommerce
# Remove title for Extensions and Themes tabs

### Submission Review Guidelines:

-   I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
-   I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
-   I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/).
-   Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:


We currently show a title and amount of results for Browse and Themes, but not for Business services. We're removing the title, this is currently repetitive as the information is already present in the tabs. It also aligns with the direction in newer WooCommerce core designs. Showing the amount of results also isn’t important, when we are showing all products.



Closes https://github.com/Automattic/woocommerce.com/issues/21567



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Check out this branch
2. Go to `/plugins/woocommerce` and run `pnpm --filter='@woocommerce/plugin-woocommerce' build`
3. Visit the [in-app marketplace](http://localhost:8888/wp-admin/admin.php?page=wc-admin&path=%2Fextensions).
4. Confirm the Extensions, Themes and Business Services tabs don't show a title
![Extensions_‹_WooCommerce_‹_WooCommerce_Core_E2E_Test_Suite](https://github.com/user-attachments/assets/fc7f9f68-f60d-4d4e-bce7-68e2381515fb)
5. Search for an extension
6. Confirm the Search tab still shows a title (the overall search experience will be updated in https://github.com/Automattic/woocommerce.com/issues/21576
![Extensions_‹_WooCommerce_‹_WooCommerce_Core_E2E_Test_Suite](https://github.com/user-attachments/assets/b03fea4a-2822-47b7-ac54-7a25975eae27)



### Changelog entry







<details>

<summary>Changelog Entry Details</summary>

#### Significance




#### Type




#### Message 
Remove titles on Extensions and Themes tabs in the marketplace
</details>

<details>

<summary>Changelog Entry Comment</summary>

#### Comment 

</details>
